src/main/webapp/script.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Event processing for upload button
+ */
+function fileUploadDisplay() {
```

Comment:
Give these functions matching names, especially `closeDiv` doesn't tell me much. Maybe `openFileUploadDialog()` and `closeFileUploadDialog()`

Reply:
Yes, I'll change those.